src/azul/service/storage_service.py

```diff
@@ -116,8 +116,8 @@ def get_presigned_url(self, key: str, file_name: Optional[str] = None) -> str:
                 **({} if file_name is None else {'ResponseContentDisposition': f'attachment;filename="{file_name}"'})
             })
 
-    def create_bucket(self, bucket_name: str = None):
-        self.client.create_bucket(Bucket=(bucket_name or self.bucket_name))
+    def create_bucket(self, bucket_name: str = None, **kwargs):
```

Comment:
Every invocation of this method uses the `kwargs` to specify the region and never uses it for any other purpose. I can't think of any circumstance where we wouldn't want to be explicit about the region, so I think the `kwargs` should be replaced with a non-optional parameter "`region`" and the construction of the `CreateBucketConfiguration` dictionary should be moved into this method.

Reply:
Even better might be to add a `region` attribute to `StorageService`, with the option to override via a parameter like with `bucket_name`, and add a `storage_service` cached property to `AzulUnitTestCase` that automatically sets the region to `self._aws_test_region`. This would however clutter up the superclass with a feature most subclasses wouldn't need, so maybe run this suggestion by Hannes first.